Sources/SwiftDocC/Infrastructure/External Data/OutOfProcessReferenceResolver.swift

```diff
@@ -175,8 +175,8 @@ public class OutOfProcessReferenceResolver: ExternalReferenceResolver, FallbackR
         return DocumentationNode(
             reference: reference,
             kind: resolvedInformation.kind,
-            sourceLanguage: .init(name: resolvedInformation.language.name, id: resolvedInformation.language.id),
-            availableSourceLanguages: Set(resolvedInformation.availableLanguages.map { .init(name: $0.name, id: $0.id) }),
+            sourceLanguage: resolvedInformation.language,
+            availableSourceLanguages: sourceLanguages(for: resolvedInformation),
```

Comment:
It looks like `sourceLanguages(for:)` also hard-codes creating a `SourceLanguage`. Should we update there as well?

```swift
private func sourceLanguages(for resolvedInformation: ResolvedInformation) -> Set<SourceLanguage> {
    if !resolvedInformation.availableLanguages.isEmpty {
        return resolvedInformation.availableLanguages
    }
    
    // Fall back to the `language` property if `availableLanguages` is empty.
    return [SourceLanguage(name: resolvedInformation.language.name, id: resolvedInformation.language.id)]
}
```

Reply:
Good catch. We should.

Reply:
Fixed in https://github.com/apple/swift-docc/pull/120/commits/9e9a3943c5a3a6c4ab580e03d628e2299d3a78f6